modules/cli/src/test/kotlin/datamaintain/cli/app/AppTest.kt

```diff
@@ -201,5 +205,18 @@ internal class AppTest : BaseCliTest() {
                 }
             }
         }
+
+        @Nested
+        inner class GenerateCompletion {
+            private val logger = LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME) as Logger
+            private val testAppender = TestAppender()
+
+            @BeforeEach
+            fun setupLogger() {
+                logger.addAppender(testAppender)
+                testAppender.start()
+            }
```

Comment:
This test class does nothing, doesn't it?

Reply:
Doh! Seems I forgot to remove that class after moving tests to a separate sub-package, sorry. I've removed it.